src/Integration/Binding/BindingController.cs

```diff
@@ -178,7 +178,8 @@ void IBindingWorkflowExecutor.BindProject(BindCommandArgs bindingArgs)
                     Language.C,
                     Language.Cpp,
                     Language.Js,
-                    Language.Ts
+                    Language.Ts,
+                    Language.Secrets
```

Comment:
Did you mean to include this change in the PR?

Reply:
Hm actually, I am not sure as I dont fully understand what the binding config provider is. Do you think it makes more sense to include it in the next pr?